pkg/github/github.go

```diff
@@ -432,7 +473,7 @@ type TagsPerBranch map[string]string
 func (g *GitHub) LatestGitHubTagsPerBranch() (TagsPerBranch, error) {
 	// List tags for all pages
 	allTags := []*github.RepositoryTag{}
-	opts := &github.ListOptions{PerPage: 100}
+	opts := &github.ListOptions{PerPage: g.Options().ItemsPerPage}
```

Comment:
Just noting here that we could have a panic if someone had manually constructed the `*Github` struct. All of our `New___` functions set the default options, so in practice this likely will not be a huge issue, but worth pointing out. One solution for areas like this are to add a `GetItemsPerPage()` method to `*Options` as Go allows you to call methods on nil structs :)

Reply:
I've added the accessor method to the options struct and changed all calls to use it :+1: